examples/oauth2/oauth2.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+
+async function doThing() {
```

Comment:
It would be cool to make this file read more properly as a consumer of our module:

Maybe rename to `example-app.js`.

Then we could add a body as follows:

```
var shapewaysApi = require("../../lib_oauth2/client_oauth2");

// Main method
async function runProgram(options) {

  // authorization code
  var client = new shapewaysApi.Client(options.clientKey, options.clientSecret);
  await client.authorize();

  // application code
  var modelData = await client.uploadModel(options.pathToModel)
}

try {
  /** 
    * require a json-encoded file with following format:
    * {
    *   clientKey: 'client_key',
    *   clientSecret: 'client_secret'
    * }
    */
  var credentials = require('./client-credentials');
  runProgram(credentials.clientKey, credentials.clientSecret);
} catch (e) {
  console.err(e);
}
```

Reply:
I like it.